### PR TITLE
Fix soname in cmake project

### PR DESCRIPTION
**Pull request by g-maxime**
Signed-off-by: Maxime Gervais <gervais.maxime@gmail.com>

### CODE REVIEW

**Comment by g-maxime** [comment]
PR updated

File: Project/CMake/CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,9 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
 set(MediaInfoLib_MAJOR_VERSION 17)
 set(MediaInfoLib_MINOR_VERSION 10)
-set(MediaInfoLib_VERSION ${MediaInfoLib_MAJOR_VERSION}.${MediaInfoLib_MINOR_VERSION})
+set(MediaInfoLib_PATCH_VERSION 0)
+set(MediaInfoLib_VERSION ${MediaInfoLib_MAJOR_VERSION}.${MediaInfoLib_MINOR_VERSION}.${MediaInfoLib_PATCH_VERSION})
```
Comment (JeromeMartinez) [inline Project/CMake/CMakeLists.txt]: Do we need the PATCH version here? I would add patch version in displayed number only for patch not 0.

**Comment by JeromeMartinez** [comment]
@Vascom please test this version.

**Comment by Vascom** [comment]
All OK.

**Comment by JeromeMartinez** [comment]
ok, I merge when Travis finishes, please update Fedora packages with that.

**Pull request merged**